theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -755,7 +759,8 @@ def test_get_rid_of_advanced_indexing_version_of_xent(self):
         for expr in expressions:
             # Verify the optimizer worked on the expressions
             f = theano.function([x, y], expr, mode=mode)
-            self.assertTrue(hasattr(f.maker.fgraph.outputs[0].tag, 'trace'))
+            assert check_stack_trace(
+                f, ops_to_check=crossentropy_softmax_argmax_1hot_with_bias)
```

Comment:
There is a `crossentropy_softmax_argmax_1hot_with_bias` with a stack trace in the graph, but only its first output has a stack trace, the other ones have an empty one.
It kind of makes sense, since only the first output is the replacement for the original variable.

I'm not sure what the right solution would be:
- add the stack trace to all outputs of the op
- select which output to check in `check_stack_trace`